src/components/AgentInput/index.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+
+class AgentInput extends Component {
```

Comment:
One of the components I created would be for this. So we’ll just need to move some of that code into this file